generators/client/templates/react/src/test/javascript/e2e/page-objects/base-component.ts.ejs

```diff
@@ -49,4 +49,8 @@ export default class BasePage {
     this.checkSelectorExist();
     waitUntilHidden(this.selector, this.constructor.name);
   }
+
+  getToast() {
+    return $('div[role="alert"]');
```

Comment:
Better to use an unique id

Reply:
I did not found out how to set ids for toast